santoku/salesforce/objects_handler.py

```diff
@@ -58,7 +58,7 @@ def __init__(
         password: str,
         client_id: str,
         client_secret: str,
-        api_version: float = 47.0,
+        api_version: str = "47.0",

```

Comment:
Why change from float to str?

Reply:
In general it is dangerous to store version numbers as floats and not strings, as version numbers are not meant to be operated with, rounded or approximated in any form.
In this instance, `api_version` is used by concatenating to a request URL via string formatting with `{:.1f}`, which would break silently if, say, we tried to use a version number with 2 "decimals".